packages/vdom/src/utils.ts

```diff
@@ -57,3 +57,11 @@ export const normalizeStyles = (styles: unknown) => {
     return result;
   }, {} as Record<string, string | number>);
 };
+
+export const changesFunc = (
```

Comment:
It looks like bad naming. By this name I don't understand what this function does.